boards/riscv/tl321x/tl3218x-common.dtsi

```diff
@@ -153,7 +153,7 @@
 };
 
 &gpiob {
-	interrupts = <37 1>;
+	interrupts = <47 1>;
```

Comment:
Why use IRQ_GPIO_IRQ1 for gpiob, Can we use IRQ_GPIO_IRQ0 for this port?

Reply:
Randomly assigned, with no restrictions

Reply:
Yes, we can assign IRQ_GPIO_IRQ0 for gpiob.
Leave IRQ_GPIO_IRQ1 to other ports like port a if we'd like to.